addons/account_peppol/models/res_partner.py

```diff
@@ -62,10 +64,14 @@ def button_account_peppol_check_partner_endpoint(self):
         """
         self.ensure_one()
 
+        peppol_param = self.env['ir.config_parameter'].sudo().get_param('account_peppol.edi.mode', False)
+        if peppol_param == 'demo':
+            self.account_peppol_is_endpoint_valid = True
```

Comment:
we probably need to set the `account_peppol_validity_last_check` here too

Reply:
yep, I actually do that in `demo_utils.py` in `mock_button_verify_partner_endpoint`
I just need to remove this condition here

Reply:
then the peppol param should not be deleted below :slightly_smiling_face: